css/main.css

```diff
@@ -6,3 +6,230 @@ Green: #00B172
 White: #FFF
 
 */
+
+* {
+    padding: 0;
+    margin: 0;
+}
+
+body {
+    font-family: 'Open Sans', sans-serif;
+}
+
+h1 {
+    font-size: 5em;
+}
+
+h2 {
+    font-size: 2em;
+}
+
+h3 {
+    font-size: 1.6em;
+}
+
+header {
+    display: flex;
+    
+    justify-content: space-between;
+    background-color: #fff;
+}
+
+.spotify-logo {
+     height: 60px;
+     padding-left: 30px;
+     padding: 20px 0px 20px 30px;
+     display: flex;
+     
+}
+
+nav {
+    position: fixed;
```

Comment:
remember that when you set the width of 100% (for example) in an element, this element will have the the width of 100% **of the parent**. Try to give to the NAV a width of 100% and white background and see what happens. :)